test/verify/check-networking-firewall

```diff
@@ -43,7 +43,7 @@ class TestFirewall(MachineCase):
         m = self.machine
 
         # HACK: default zone has no interface in permanent configuration
-        if m.image in ["debian-stable"]:
+        if m.image in ["debian-stable", "debian-testing"]:
```

Comment:
Is this a rebase error? It's apparently not done  for the other tests, and if it's intended it should be in the previous commit.